dm/dumpling/dumpling_test.go

```diff
@@ -124,3 +125,32 @@ func (d *testDumplingSuite) TestDefaultConfig(c *C) {
 	c.Assert(dumpling.dumpConfig.StatementSize, Not(Equals), export.UnspecifiedSize)
 	c.Assert(dumpling.dumpConfig.Rows, Not(Equals), export.UnspecifiedSize)
 }
+
+func (d *testDumplingSuite) TestCallStatus(c *C) {
+	conf := export.DefaultConfig()
+	cfg := &config.SubTaskConfig{}
+	m := &Dumpling{cfg: cfg}
+	ctx := context.Background()
+
+	ss := &binlog.SourceStatus{}
+	s := m.Status(ss).(*pb.DumpStatus)
+	c.Assert(s.CompletedTables, Equals, float64(0))
+	c.Assert(s.FinishedBytes, Equals, float64(0))
+	c.Assert(s.FinishedRows, Equals, float64(0))
+	c.Assert(s.EstimateTotalRows, Equals, float64(0))
+
+	dumpling, err := export.NewDumper(ctx, conf)
+	if err != nil {
```

Comment:
we should use `c.Assert` to check err is nil, then assign `m.realdumper`

BTW, I also like the name `m.core` as recommended in https://github.com/pingcap/ticdc/pull/3694/files#r761572067